numba_dpex/core/targets/kernel_target.py

```diff
@@ -111,12 +111,15 @@ def load_additional_registries(self):
         """Register the OpenCL API and math and other functions."""
         from numba.core.typing import cmathdecl, enumdecl, npydecl
 
+        from numba_dpex.core.typing import dpnpdecl
+
         from ...ocl import mathdecl, ocldecl
 
         self.install_registry(ocldecl.registry)
         self.install_registry(mathdecl.registry)
         self.install_registry(cmathdecl.registry)
         self.install_registry(npydecl.registry)
```

Comment:
Do we still need `npydecl` to be registered?

Reply:
If we are able to get rid of `npydecl` it should resolve #146. Whether we should allow `dpnp` calls in kernel is another matter.

Reply:
Apparently #146 was resolved in #1267 . It broken dpbench workflow that was calling numpy function in kernel

Reply:
We cannot get rid of because of the `truediv` workaround. We need to investigate it deeper to be able generate dpnp specific ufunc without calling `numpy.truediv` function. It is out of scope of this PR, so no changes needed.

Reply:
Thanks for the explanation. Can you add a comment with the explanation on the line where the `npydecl.registry` is installed?